akka-serializability-checker-compiler-plugin/src/main/scala/org/virtuslab/akkasaferserializer/AkkaSerializabilityCheckerCompilerPlugin.scala

```diff
@@ -3,16 +3,16 @@ package org.virtuslab.akkasaferserializer
 import scala.tools.nsc.Global
 import scala.tools.nsc.plugins.{Plugin, PluginComponent}
 
-class AkkaSerializabilityCheckerPlugin(override val global: Global) extends Plugin {
+class AkkaSerializabilityCheckerCompilerPlugin(override val global: Global) extends Plugin {
```

Comment:
Hmmm everything in this project relates to Akka by default... e.g. there's just `dump schema event`, no mention of Akka.

Reply:
So I'd remove the `Akka` prefix unless you see a reason to leave it here specifically